microsoftgraph/docs-conceptual/tutorial-grant-delegated-api-permissions.md

```diff
@@ -0,0 +1,166 @@
+    ```
+
+    Select **Consent on behalf of your organization** before accepting in the login dialog box.
+
+>[!Caution]
+>The `DelegatedPermissionGrant.ReadWrite.All` permission allows an app or a service to manage permission grants and elevate privileges for any app, user, or group in your organization. Only appropriate users should access apps that have been granted this permission.
+
+## Step 1: Get the delegated permissions of the resource service principal
+
+Before you can grant delegated permissions, you must first identify the delegated permissions to grant and the resource service principal that exposes the delegated permissions. Delegated permissions are defined in the `oauth2PermissionScopes` object of a service principal. 
+
+In this article, you'll use the `Microsoft Graph` service principal in the tenant as your resource service principal.
+
+```powershell
+Get-MgServicePrincipal -Filter "displayName eq 'Microsoft Graph'" -Property Oauth2PermissionScopes | Select -ExpandProperty Oauth2PermissionScopes | fl 
+```
+
+```Output
+AdminConsentDescription : Allows the app to read and write the full set of profile properties, reports, and managers of other users in your organization, on behalf of the signed-in user.
+AdminConsentDisplayName : Read and write all users' full profiles
+Id                      : 204e0828-b5ca-4ad8-b9f3-f32a958e7cc4
+IsEnabled               : True
+Origin                  :
+Type                    : Admin
+UserConsentDescription  : Allows the app to read and write the full set of profile properties, reports, and managers of other users in your organization, on your behalf.
+UserConsentDisplayName  : Read and write all users' full profiles
+Value                   : User.ReadWrite.All
+AdditionalProperties    : {}
+
+AdminConsentDescription : Allows the app to list groups, and to read their properties and all group memberships on behalf of the signed-in user.  Also allows the app to read calendar, conversations, files,
+                          and other group content for all groups the signed-in user can access.
+AdminConsentDisplayName : Read all groups
+Id                      : 5f8c59db-677d-491f-a6b8-5f174b11ec1d
+IsEnabled               : True
+Origin                  :
+Type                    : Admin
+UserConsentDescription  : Allows the app to list groups, and to read their properties and all group memberships on your behalf.  Also allows the app to read calendar, conversations, files, and other group
+                          content for all groups you can access.
+UserConsentDisplayName  : Read all groups
+Value                   : Group.Read.All
+AdditionalProperties    : {}
+```
+
+## Step 2: Create a client service principal
+
+The first step in granting consent is to [create the service principal](/powershell/module/microsoft.graph.applications/new-mgserviceprincipal?view=graph-powershell-1.0&preserve-view=true). To do so, you'll need the `App Id` of your application.
```

Comment:
Same - "[create a service principal] for the app that you'll grant permissions."